RELEASE/scripts/autoscend/auto_buff.ash

```diff
@@ -142,7 +142,7 @@ boolean buffMaintain(effect buff, int mp_min, int casts, int turns, boolean spec
 	case $effect[Ashen]:					useItem = $item[pile of ashes];						break;
 	case $effect[Ashen Burps]:					useItem = $item[ash soda];						break;
 	case $effect[Astral Shell]:
-		if(acquireTotem())
```

Comment:
adding auto_have_skill to buffmaintain lookup list is redundant with the changes you made to auto_acquire.ash
you can remove all instances of it.